explorer/client/src/utils/static/owned_object.json

```diff
@@ -6,6 +6,80 @@
                 {
                     "objectId": "playerOne"
                 },
+                {
+                    "objectId": "anotherPlayer"
```

Comment:
also need to vary the duplicate mocked object IDs here, since object IDs are unique

Reply:
This is legacy code I've now removed.